src/main-client.ts

```diff
@@ -337,27 +631,87 @@ export class MainClient extends BaseRestClient {
     return this.getPrivate('sapi/v1/account/apiRestrictions');
   }
 
-  acceptQuoteRequest(params: AcceptQuoteRequestParams): Promise<any> {
-    return this.postPrivate('sapi/v1/convert/acceptQuote', params);
-  }
-
   enableConvertSubAccount(params: EnableConvertSubAccountParams): Promise<any> {
     return this.postPrivate('sapi/v1/broker/subAccount/convert', params);
   }
 
-  convertQuoteRequest(params: ConvertQuoteRequestParams): Promise<any> {
-    return this.postPrivate('sapi/v1/convert/getQuote', params);
+  getFundingAsset(params: GetAssetParams): Promise<FundingAsset[]> {
+    return this.postPrivate('sapi/v1/asset/get-funding-asset', params);
   }
 
-  getOrderStatus(params: GetOrderStatusParams): Promise<any> {
-    return this.getPrivate('sapi/v1/convert/orderStatus', params);
+  getUserAsset(params: GetAssetParams): Promise<UserAsset[]> {
+    return this.postPrivate('sapi/v3/asset/getUserAsset', params);
   }
 
-  getConvertTradeHistory(params: GetConvertTradeHistoryParams): Promise<any> {
-    return this.getPrivate('sapi/v1/convert/tradeFlow', params);
+  convertBUSD(params: ConvertTransferParams): Promise<ConvertTransferResponse> {
+    return this.postPrivate('sapi/v1/asset/convert-transfer', params);
+  }
+
+  getConvertBUSDHistory(
+    params: GetConvertBUSDHistoryParams,
+  ): Promise<GetConvertBUSDHistoryResponse> {
```

Comment:
A bit nitpicky but I think it would be more tidy. Right now, the return type is something like this:
```
export interface GetConvertBUSDHistoryResponse {
  total: number;
  rows: BUSDConversionRecord[];
}
```

I think it's more tidy to have this instead:
```
  getConvertBUSDHistory(
    params: GetConvertBUSDHistoryParams,
  ): Promise<{ total: number; rows: BUSDConversionRecord[]; }> {
```

This is the interface used in both:
```
export interface BUSDConversionRecord {
  tranId: number;
  type: number;
  time: number;
  deductedAsset: string;
  deductedAmount: string;
  targetAsset: string;
  targetAmount: string;
  status: string;
  accountType: string;
}
```

Might be more repetitive with many methods having this total + rows pattern, but I think it helps readability.